web/src/main/java/org/akaza/openclinica/control/submit/DownloadAttachedFileServlet.java

```diff
@@ -119,7 +119,8 @@ public void processRequest() throws Exception {
             addPageMessage("File " + filePathName + " " + respage.getString("not_exist"));
             
             // try to use the passed in the existing file
-        	file = new File(fileName);
+            //OC-17868
+        	//file = new File(fileName);

```

Comment:
Is this code not needed? It looks like it was added to resolve https://jira.openclinica.com/browse/OC-12392. Can you verify that https://jira.openclinica.com/browse/OC-12392 still works without this line? If it does, can you remove it completely instead of commenting it out?

Reply:
code changed --Did testing both OC-12392 and OC-17868, now both fixed